manifests/overlays/kubeflow/kustomization.yaml

```diff
@@ -3,8 +3,6 @@ kind: Kustomization
 namespace: kubeflow
 resources:
   - ../../base
-  - ../../manager
-  - ../../rbac
   - kubeflow-training-roles.yaml
 images:
   - name: kubeflow/training-operator:v1.0.0
```

Comment:
Should we keep `newName: kubeflow/training-operator` instead of  `newName: deepakmuley/kubeflow-training-operator` here for now ?

Reply:
I may miss that file. Let me check it

Reply:
nice catch! I forget to update it. I push a new image in docker hub at this moment. Once post-submit job is fixed. We will prepare a new image & tag and update it for release.

Reply:
@andreyvelich Please have another check